src/commands/download_drive.ts

```diff
@@ -42,7 +42,9 @@ new CLICommand({
 				wallet,
 				arweave
 			});
-			const driveKey = await parameters.getDriveKey({ driveId });
+
+			const driveSignatureInfo = await ardrive.getDriveSignatureInfo(driveId, await wallet.getAddress());
+			const driveKey = await parameters.getDriveKey({ driveId, driveSignatureInfo });
```

Comment:
Small nit that this pattern is repeated so often it likely deserves its own utility function. One that either accepts parametershelper and an ardrive instance and a driveId or simply an expansion of the existing parametershelper method to accept an ardrive instance. Something to that effect.

Reply:
Updated in 2b69935aaa96b2212700ede94caab5594d756585